content/v1.12/concepts/managed-resources.md

```diff
@@ -3,630 +3,745 @@ title: Managed Resources
 weight: 102
```

Comment:
meta: the right hand "on this page" TOC is long enough to scroll by default, but there's no obvious visual indication that it can be scrolled.  any improvements you've already been thinking about making there?

Reply:
🤔 what browser? There should be a scrollbar
![Screenshot 2023-06-27 at 12 53 33 PM](https://github.com/crossplane/docs/assets/10537576/5b78dc1b-c8df-431f-9aea-6cf1c22a73e9)